src/www/app/vpn_installer.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Helper class to install VPN services to the OS.
+ */
+export interface VpnInstaller {
+  /**
+   * Setup the underlying outline VPN services.
+   */
+  setupVpn(): Promise<void>;
```

Comment:
Please align the wording.

If this is called `VpnInstaller`, then the mehod should be `installVpn()`.

Or maybe we can call it `VpnService` and just have a `setUp/install()` method?

I think we could make it part of the RoutingService if we changed its scope, but that would be a bigger change.